pandas/extensions/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Public API for extending panadas objects."""
```

Comment:
this is better, but still not a fan of extending the top-level namespaces (e.g. with extensions). api.extensions would be better